### ISSUE TITLE
Markdown print export (feature request)

### ISSUE THREAD

**Issue by dluxcru** [chummer5a/chummer5a#1317]
(Please fill out the below as accurately as possible. More detail makes it easier for us to find the problems and fix them.)

### Environment
Chummer Version:  5.188.3
Environment:  Windows 10
Runtime: (If you don't know this, it is probably .NET 4.5.1)

### Expected behavior
Would like a print export format that allow you to save in Markdown, rather than HTML. This would be valueable for exporting characters to Obsidian Portal. 

### Actual behavior
(Enter text here)

### Steps to reproduce the behavior
(Enter text here)

### Link to affected character file
(Enter text here)

**Comment by chummer5a** [comment]
Hrm. If you can give me a mapping of common slugs that are used I can do an export easily enough, I'm not particularly interested in doing a pre-rendered output beyond that though.

**Comment by dluxcru** [comment]
I wouldn't expect rendered - simply having a text format with Markdown slugs would be good enough.  Is this what you're looking for? http://daringfireball.net/projects/markdown/syntax

**Comment by chummer5a** [comment]
Nevermind, I'd forgotten how Obsidian Portal manages character sheets and thought it was a user-accessible set of values. There *might* be a way I can do it easily, but the easiest method is probably going to be copying a HTML sheet into [PanDoc](https://pandoc.org/try/) or something.

**Comment by dluxcru** [comment]
I already tried that (I am a programmer as well, utilities like PanDoc aren't new to me) and it didn't produce much useful output. The text-only format works decently well, though - You just need to deal with the quantities of items in text-only fields when they've been added as plugins not showing up, which I mentioned in one of the other errors I reported. 

(Modifying text only or formatted text only to include markdown in the right spots when a checkbox is selected in the right spots might be an easy way to go - would just make thing more readable. )
